FABulous/FABulous.py

```diff
@@ -1,3 +1,9 @@
+"""FABulous command-line interface entry point.
+
+This module provides the main entry point for the FABulous FPGA toolkit command-line
```

Comment:
Did this slip through in my last review or did you leave it in on purpose?
```suggestion
This module provides the main entry point for the FABulous FPGA framework command-line
```